docs/userGuide/syntax/annotations.md

```diff
@@ -0,0 +1,145 @@
+</annotate>
+</variable>
+</include>
+
+**Using trigger for Annotate Point** <br>
+
+Similar to **[popovers](popups.md)**, annotation points also support different types of triggers
+for users with different needs.
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">html</variable>
+<variable name="code">
+
+<annotate src="../../images/annotateSampleImage.png" width="500" alt="sampleImage">
+  <!-- Default Trigger (click)-->
+  <a-point x="33%" y="33%" content="Lorem ipsum dolor sit amet" />
+  <!-- Set Trigger to hover focus -->
+  <a-point x="66%" y="33%" content="Lorem ipsum dolor sit amet" trigger="hover focus"/>
+  <!-- Set Popover Placement (click)-->
+  <a-point x="25%" y="66%" content="Lorem ipsum dolor sit amet" placement="left"/>
+  <a-point x="50%" y="66%" content="Lorem ipsum dolor sit amet" placement="bottom"/>
+  <a-point x="75%" y="66%" content="Lorem ipsum dolor sit amet" placement="right"/>
+</annotate>
+</variable>
+</include>
+
+<br>
+
+**Displaying content as legends in Annotate Point** <br>
+
+Annotate point also allows users to display its content inside the popover or as
+a legend below the diagram or both. However, this only works when if a label is
+also provided for the point.
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">html</variable>
+<variable name="code">
+
+<annotate src="../../images/annotateSampleImage.png" width="500" alt="sampleImage">
+  <!-- Default Legend (popover only)-->
+  <a-point x="25%" y="50%" content="There is only text when you click me" label="1"/>
+  <!-- Set Legend to bottom only (popover is not clickable) -->
+  <a-point x="50%" y="50%" content="Clicking on this does nothing" label="2" legend="bottom" header="Headers are displayed as well"/>
+  <!-- Set Legend to both -->
+  <a-point x="75%" y="50%" content="There is text at both locations"  label="3" legend="both" header="Headers are displayed at both positions"/>
+</annotate>
+</variable>
+</include>
+
+<br>
+
+****`<a-point>` Options****
+
+| Name      | Type     | Default     | Description                                                                                                                              |
+| --------- | -------- | ----------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| x         | `String` | `''`        | **This must be specified.**<br>How to position the x-coordinate of the point.<br>Supports range of values from `0%` to `100%`.           |
+| y         | `String` | `''`        | **This must be specified.**<br>How to position the y-coordinate of the point.<br>Supports range of values from `0%` to `100%`.           |
+| content   | `String` | `''`        | **This must be specified.**<br>The content to be inserted into the body of the text.<br>The body will be omitted if this is not provided |
```

Comment:
If they have a default known to the user, then it may not be true that "This must be specified"? Especially for `content`

Reply:
Hmm that's true, they could get away with omitting the content if they wished. Perhaps I should allow it... Any ideas on how to improve this?

Reply:
I think as a user I want to know if it is required or can be omitted totally (by not specifying the attribute), so perhaps can reword to mention whether it's required. Also can consider removing the "How to position" to be more concise

Reply:
Ok based on the feedback I made x and y have no default value and made content optional!